src/task_8/task_8_3.py

```diff
@@ -0,0 +1,38 @@
+"""
+Описать функцию Sin1( x , ε ) вещественного типа (параметры x , ε —
+вещественные, ε > 0), находящую приближенное значение функции sin( x ):
+sin( x ) = x – x ^3 /(3!) + x^ 5 /(5!) – ... + (–1) ^ n · x^( 2·n+1) /((2· n +1)!) + ... .
+В сумме учитывать все слагаемые, модуль которых больше ε . С помощью
+Sin1 найти приближенное значение синуса для данного x при шести данных
+ε.
+"""
+import math
+def sin1(x: float, e: list) -> float:
+    """
+    Функция по расчету синуса угла в радианах.
+    :param x: float. значение, по которому вычисляется синус (в радианах)
+    :param e > 0: list. (список значений точности, по которым высчитыввется приближенное значение синуса)
```

Comment:
В е будет содержаться не список значение, а только одно значение

```suggestion
    :param e > 0: float. (список значений точности, по которым высчитыввется приближенное значение синуса)
```